src/platform/iotjs_module_uart-linux-general.inl.h

```diff
@@ -135,6 +135,14 @@ void OpenWorkerUart(uv_work_t* work_req) {
 }
 
 
+void UartClose() {
+  uv_close((uv_handle_t*)&poll_handle, NULL);
+  if (fd > 0) {
```

Comment:
What do you mean by `fd > 0`? Valid file descriptor is non-negative number and zero is standard input. Therefore you are trying to close all seemingly valid file descriptor except for stdin. What is going about stdout and stderr? If you mean all valid file descriptor, it should be (fd >= 0).